phpBB/phpbb/template/twig/twig.php

```diff
@@ -344,11 +353,20 @@ protected function get_template_vars()
 			$context_vars['.'][0], // To get normal vars
 			array(
 				'definition'	=> new \phpbb\template\twig\definition(),
-				'user'			=> $this->user,
 				'loops'			=> $context_vars, // To get loops
 			)
 		);
 
+		if ($this->user instanceof \phpbb\user)
+		{
+			$vars = array_merge(
```

Comment:
My intention was to be able to keep `$context_vars['.'][0]['user']['something']` if that is something that is used. However it is not required I fix it